### PR TITLE
Refactor core files

### PR DESCRIPTION
**Pull request by kagikn**
This will improve readability without serious performance issues 😊

### CODE REVIEW

File: source/core/Console.cs [hunk 1]
```diff
@@ -34,107 +34,43 @@ namespace GTA
 	//Used to describe console args (Needed for creating a Help-func)
 	internal class ConsoleArg
 	{
-		#region Fields
-		string _type;
-		string _name;
-		#endregion
-
 		public ConsoleArg(string type, string name)
```
Comment (crosire) [inline source/core/Console.cs]: We could get rid of this constructor altogether and make the auto properties below settable.

Comment (kagikn) [reply source/core/Console.cs]: If we do that, we have to write like this, don't we?
```
	var arg = new ConsoleArg();
	arg.Type = args.ParameterType.Name;
	arg.Name = args.ParameterType.Name;

	command.ConsoleArgs.Add(arg);
```

Comment (crosire) [reply source/core/Console.cs]: Ignore my previous comment. I mixed up the class with the console command attribute. Doesn't make sense to have setters on this one actually.

Comment (kagikn) [reply source/core/Console.cs]: Okay, reverted the commit.

**Pull request merged**